styles/_blog.less

```diff
@@ -202,6 +206,14 @@ body:not(.button-style-default) .sqs-editable-button, body:not(.button-style-def
   border-bottom: 2px solid @badgerRed;
 }
 
+.sqs-title:hover {
+  display: inline;
+}
+
+.sqs-title:hover ~ .sqs-content {
+  margin-top: 14px;
+}
+
```

Comment:
Sorry for that ugly hack but the squarespace slices the title surrounding it parts in `<em>` tags so there is no other way around it 🤖